src/query/graphite/storage/converter_test.go

```diff
@@ -45,6 +45,20 @@ func TestConvertMetricPartToMatcher(t *testing.T) {
 	}
 }
 
+func TestConverWildcardToMatcher(t *testing.T) {
+	metric := "*"
+	for i := 0; i < 100; i++ {
```

Comment:
This is kind of a weird test -- why not just run this once?

Reply:
Just to ensure that the metric name is also converted to expected (i.e. `__g100__`) as well